frontend/src/lib/components/AddToDashboard/AddToDashboard.tsx

```diff
@@ -21,7 +22,7 @@ export function AddToDashboard({ insight, setOpenModal }: SaveToDashboardProps):
                 type="secondary"
                 icon={
                     <IconWithCount count={dashboards.length} showZero={false}>
-                        <IconGauge />
+                        <IconDashboard />
```

Comment:
feels like a strange renaming... is this still the gauge icon but it's called "dashboard"?

Reply:
Yeah. Just moved to the equivalent in the new `@posthog/icons` repo. The naming of icons is a little different.

Still represents a gauge but looks slightly different. Diff is https://github.com/PostHog/posthog/pull/20528/files#diff-cc555de7029b6c08dbab3eb5cfde9d57da0c653b3ead7cabebcb8c663d61a922